Tools/MonoGame.Effect.Compiler/Program.cs

```diff
@@ -249,7 +249,7 @@ private class ConsoleEffectCompilerOutput : IEffectCompilerOutput
         {
             public void WriteWarning(string file, int line, int column, string message)
             {
-                Console.WriteLine("{0}({1},{2}): warning PREPROCESS01: {3}", ConvertToNative(file), line, column, ConvertMessage(message));
+                Console.Error.WriteLine("{0}({1},{2}): warning PREPROCESS01: {3}", ConvertToNative(file), line, column, ConvertMessage(message));
```

Comment:
MGCB does not show the Console.WriteLine (aka stdout) when it calls MGFXC to compile shaders.  It only shows the `stderr` output.  This ensures that any warnings from the FX preprocessor appear in the MGCB output.